src/pkg/bundle/tui/deploy/model.go

```diff
@@ -64,7 +64,6 @@ type pkgState struct {
 	percDownloaded     int
 	downloaded         bool
 	verified           bool
-	isRemote           bool
```

Comment:
I think this may break some TUI things?

Reply:
yeah, this was weird, not sure if it was a merge thing, but the `isRemote` field was making my linter complain that it isnt being used anywhere